xlators/mgmt/glusterd/src/glusterd.c

```diff
@@ -1666,14 +1620,6 @@ init(xlator_t *this)
            "directory",
            rundir);
 
-    ret = glusterd_find_correct_var_run_dir(this, var_run_dir);
-    if (ret) {
-        gf_msg(this->name, GF_LOG_CRITICAL, 0, GD_MSG_VAR_RUN_DIR_FIND_FAIL,
-               "Unable to find "
-               "the correct var run dir");
-        exit(1);
-    }
-
     ret = glusterd_init_var_run_dirs(this, var_run_dir,
```

Comment:
?

Reply:
We only ever use `/run` so there’s no need of this function. Should I remove the `GD_MSG_VAR_RUN_DIR_FIND_FAIL` as well?

Reply:
I'm in favor of leaving the mesg, it can only mess IDs if you don't do it right. But should this function be removed or renamed? What's under all those /var/run/{bitd|quotad|...} ?